### PR TITLE
Update serving/_index.md with network layer option

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Proposed Changes

- [#3117] Add ref for contour

### CODE REVIEW

File: docs/serving/_index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The Knative Serving project provides middleware components that enable:
 
 - Rapid deployment of serverless containers.
 - Autoscaling including scaling pods down to zero.
-- Support for multiple networking layers such as Ambassador, Courier, Gloo, Istio, and Kong, for integration into existing environments.
+- Support for multiple networking layers such as Ambassador, Contour, Courier, Gloo, Istio, and Kong, for integration into existing environments.
```
Comment (nak3) [inline docs/serving/_index.md:14]: In addition to this change, could you please also fix `Courier` to [`Kourier`](https://github.com/knative-sandbox/net-kourier#kourier) ?

Comment (abrennan89) [inline docs/serving/_index.md:14]:
```suggestion
- Support for multiple networking layers such as Ambassador, Contour, Kourier, Gloo, Istio, and Kong, for integration into existing environments.
```

Comment (xtreme-sameer-vohra) [reply docs/serving/_index.md]: Ah good catch :)

**Pull request merged**